build-debug/CMakeCache.txt

```diff
@@ -0,0 +1,375 @@
+# This is the CMakeCache file.
```

Comment:
This is my bad, I forgot to add `build-*` to the `.gitignore`.

In general, you should avoid doing `git add .` or related commands, though - instead prefer to add individual files you want to change.